src/agent/default/prompt.py

```
@@ -0,0 +1,8 @@
+from pydantic import BaseModel, validate_call
```

Ruff (0.8.2)

`pydantic.validate_call` imported but unused

Remove unused import: `pydantic.validate_call`

(F401)